.github/workflows/elixir-build-check.yaml

```diff
@@ -86,3 +86,22 @@ jobs:
           mix-env: ${{ inputs.mix-env }}
       - name: Check for circular dependencies
         run: mix xref graph --label compile-connected --fail-above 0
+  format:
+    name: Validate formatting complies with `mix format`
+    needs: [dependencies]
+    runs-on: ${{ inputs.runs-on }}
+    steps:
+      - name: Fetch the full repository so we can perform a Git diff
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Fetch dependencies as they may be needed by the formatter
+        uses: rentpath/actions-services-elixir-deps@v1
+        with:
+          elixir-version: ${{ inputs.elixir-version }}
+          erlang-version: ${{ inputs.erlang-version }}
+          mix-env: ${{ inputs.mix-env }}
```

Comment:
Dependencies MAY be needed by the `mix format` command as formatting rules will be pulled from any dependencies listed in the `import_deps` option in `.formatter.exs`. See  https://hexdocs.pm/mix/1.12.3/Mix.Tasks.Format.html#module-importing-dependencies-configuration for details.